### PR TITLE
Write Webapp E2E with Cypress: "MM-T2042 Full Name starting blank stays blank"

### PR DESCRIPTION
**Pull request by sinansonmez**
#### Summary
This PR adds Webapp E2E test with Cypress for "MM-T2042 Full Name starting blank stays blank"

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/18824

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Comment by sinansonmez** [comment]
> I also saw in JIRA that there will be an effort to migrate E2E tests to typescript. If the groundwork is ready, I can convert this file to typescript. Please let me know.

Same comment here

File: e2e/cypress/tests/integration/account_settings/profile/fullname_edit_spec.js [hunk 1]
```diff
@@ -39,5 +38,48 @@ describe('Profile > Profile Settings > Full Name', () => {
...
+        // # Click "Edit" to the right of "Full Name"
+        cy.get('#nameEdit').should('be.visible').click();
+
+        // # Save the settings
+        cy.uiSave();
+
+        // * Check that the full name was correctly cleared
+        cy.findByText("Click 'Edit' to add your full name").should(
+            'be.visible',
+        );
```
Comment (josephbaylon) [inline e2e/cypress/tests/integration/account_settings/profile/fullname_edit_spec.js:71]: Please write this in one line.

**Review by josephbaylon — APPROVED** [review]
@sinansonmez Thanks for your contribution!
LGTM just a minor change request below.

**Review by mvitale1989 — APPROVED** [review]
@sinansonmez thanks for your contribution, also looks good to me

**Pull request merged**